### DISCUSSION TITLE
How do I enable pasting figures in qmd files with VS Code?

### DISCUSSION THREAD

**Discussion by anupamsobti** [quarto-dev/quarto-cli#8014] [Q&A] [answered]
### Description

I'm using Quarto for making a presentation and I often need to paste figures from screenshots/the web. VS Code seems to now support this natively for markdown files (https://stackoverflow.com/questions/75831497/how-can-i-paste-images-into-my-markdown-files-in-vs-code). How do I enable this for quarto markdown?

**Comment by mcanouil** [comment, marked as answer]
You can't currently, this is a feature request to do for the VSCode extension at <https://github.com/quarto-dev/quarto>.

Note that if you change the document type to markdown VSCode can do what you are looking for.

**Reply by anupamsobti** [reply]
Thanks. I did the latter (instructions [here](https://stackoverflow.com/questions/29973619/how-to-associate-a-file-extension-with-a-certain-language-in-vs-code/51228725#51228725)) and it works perfectly!
